### DISCUSSION TITLE
Include Bangladesh in Sponsors Program

### DISCUSSION THREAD

**Discussion by fluentmoheshwar** [community/community#67177] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

There are many awesome developers in Bangladesh. I would appreciate if Bangladesh was included in the GitHub Sponsors Program

**Comment by anjuan** [comment, marked as answer]
Hi @fluentmoheshwar, I'm the Engineering Manager for GitHub Sponsors. We're always looking to expand Sponsors into new areas, and we have to consider a number of logistical factors for where we take the program. I'd love to offer Sponsors to the awesome developers in Bangladesh, but I can't commit to a timeline at this time. However, I will definitely take this back to the team!
